kernels/volk/volk_32f_log2_32f.h

```diff
@@ -403,7 +397,8 @@ volk_32f_log2_32f_u_generic(float* bVector, const float* aVector, unsigned int n
   unsigned int number = 0;
 
   for(number = 0; number < num_points; number++){
-    *bPtr++ = log2f(*aPtr++);
+    float const result = log2f(*aPtr++);
```

Comment:
I'd advise to create a macro that does these 2 lines, then the macro can be called from both here and the `generic` function. That way any change to the macro affects everything; just have to make 1 change for all kernels to benefit!

Reply:
All SIMD kernels call the generic kernel, but anyway it is good to make it clear that this implementation is not IEEE FP conform.

Reply:
Looks good!